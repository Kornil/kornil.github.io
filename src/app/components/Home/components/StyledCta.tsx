import styled, { StyledComponent } from "react-emotion";

interface StyledCtaInterface {
  [key: string]: StyledComponent<{}, any, any>;
}

const StyledCta: StyledCtaInterface = {
  Container: styled.section`
    align-self: center;
    justify-self: center;
  `,
  Header: styled.h1`
    margin-top: 0;
  `
};

export default StyledCta;
