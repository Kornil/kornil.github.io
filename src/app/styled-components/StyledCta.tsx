import styled, { StyledComponent } from "react-emotion";

interface StyledCtaInterface {
  [key: string]: StyledComponent<{}, any, any>;
}

const StyledCta: StyledCtaInterface = {
  Container: styled.section``,
  Header: styled.h1``
};

export default StyledCta;
