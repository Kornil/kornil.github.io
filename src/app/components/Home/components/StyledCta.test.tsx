import { shallow } from "enzyme";
import React from "react";

import StyledCta from "./StyledCta";

test("it works", () => {
  for (const key in StyledCta) {
    const Component = StyledCta[key];
    const wrapper = shallow(<Component />);
    expect(wrapper).toMatchSnapshot();
  }
});
