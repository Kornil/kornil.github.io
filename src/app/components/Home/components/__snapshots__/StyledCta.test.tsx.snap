// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.emotion-0 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  justify-self: center;
}

<section
  className="emotion-0 emotion-1"
/>
`;

exports[`it works 2`] = `
.emotion-0 {
  margin-top: 0;
}

<h1
  className="emotion-0 emotion-1"
/>
`;
